### ISSUE TITLE
fake error

### ISSUE THREAD

**Issue by larabred14-star** [validator/validator#2086]
### Summary

this doesn't make any sense, it's the endof the website and it has nothing more than the privacy poicy and design by, there shouldn't be any errors

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<div class="grelha max ">
    <div id="line">
      <h6>Caretos de Podence © 2026. Todos os direitos reservados.</h6>
      <a href="#">
        <h6>Política de Privacidade</h6>
      </a>
      <a href="#">
        <h6>Condições Gerais</h6>
      </a>
    </div>

    <h6 id="spacegirl">Design Lara Costa</h6>
  </div>
```

### Checker error messages

```shell
The heading h6 (with computed level 6) follows the heading h2 (with computed level 2), skipping 3 heading levels.
```

### Steps to reproduce

1.copy code snippet
2.paste in validator
3.validate

### Expected behavior

there shouldn't be any errors

### Actual behavior

it says theres an error

**Comment by j9t** [comment]
Hi @larabred14-star, do you have more context?

The code snippet alone does not suffice to reproduce. When adding a doctype and a title (as is needed in a valid HTML document), the snippet you shared is fine (paste [in the validator](https://validator.w3.org/nu/#textarea)):

```html
<!doctype html>
<title>Test</title>

<div class="grelha max ">
    <div id="line">
      <h6>Caretos de Podence © 2026. Todos os direitos reservados.</h6>
      <a href="#">
        <h6>Política de Privacidade</h6>
      </a>
      <a href="#">
        <h6>Condições Gerais</h6>
      </a>
    </div>

    <h6 id="spacegirl">Design Lara Costa</h6>
  </div>
```

The message seems to relate to other headings in the document. If you can share more, I (or someone else) could review and either advise or update the validator.

**Comment by sideshowbarker** [comment]
Closing for lack of response
